vonk/plugins/plugins_order.rst

```diff
@@ -0,0 +1,86 @@
+.. _vonk_plugins_order:
+
+The order of plugins
+====================
+
+The :ref:`VonkConfiguration attribute <vonk_reference_api_pipeline_configuration>` lets you define an ``Order``. This page explains how to choose a suitable number for that order.
+
+.. _vonk_plugins_order_inspect:
+
+Inspect order numbers in use
+----------------------------
+
+If you start Vonk, the log lists all the loaded plugins, with their order. You can see an example :ref:`here <vonk_plugins_log_pipeline>`. Also the list of :ref:`vonk_available_plugins` includes the order number chosen for each of those plugins.
+
+.. _vonk_plugins_order_minimum:
+
+Minimum order
+-------------
+
+Registering services
+^^^^^^^^^^^^^^^^^^^^
+
+If you only need to register services for the dependency injection framework, the order is only relevant if you need to override a registration done by Vonk. There are two ways:
```

Comment:
How do I know if I only need to register services for the dependency framework? (real question)

Reply:
Added extra explanation on that.